packages/adapters-library/src/adapters/aave-v2/common/aaveBasePoolAdapter.ts

```diff
@@ -176,18 +176,6 @@ export abstract class AaveBasePoolAdapter implements IProtocolAdapter {
     const metadataObject: ProtocolToken[] = []
 
     const promises = reserveTokens.map(async ({ tokenAddress }) => {
-      const reserveConfigurationData =
-        await protocolDataProviderContract.getReserveConfigurationData(
-          tokenAddress,
-        )
-
-      if (
-        !reserveConfigurationData.isActive ||
-        reserveConfigurationData.isFrozen
-      ) {
-        return
-      }
```

Comment:
So... we want to show inactive and frozen pools?

I don't remember why we added this in the first place. Is there any example that wasn't showing before?